.github/ISSUE_TEMPLATE/RELEASE.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: release
+assignees: ''
+
+---
+
+- [ ] Start a new release branch:
+```bash
+$ git flow release start X.Y.Z
```

Comment:
After discussing with Scott, we determined that our typical Git Flow workflow would be most comfortable here.